packages/grid/x-data-grid-premium/src/hooks/features/aggregation/wrapColumnWithAggregation.tsx

```diff
@@ -131,22 +131,38 @@ const getWrappedFilterOperators: ColumnPropertyWrapper<'filterOperators'> = ({
   getCellAggregationResult,
 }) =>
   filterOperators!.map((operator) => {
+    const baseGetApplyFilterFn = operator.getApplyFilterFn;
+
+    const getApplyFilterFn: GridFilterOperator<any, any, any>['getApplyFilterFn'] =
+      baseGetApplyFilterFn.v7
```

Comment:
This looks outdated, since we don't have the v7 flag on `getApplyFilterFn` anymore